client/src/pages/reports/Show.js

```diff
@@ -449,6 +451,7 @@ class ReportShow extends Page {
 				<LinkTo person={person} />
 			</td>
 			<td><LinkTo position={person.position} /></td>
```

Comment:
When a person has no position it will show unspecified for position, maybe it's better to add whenUnspecified="" ?